.travis.yml

```diff
@@ -10,3 +10,6 @@ before_install:
 script:
   - bundle exec rake test
   - bundle exec rubocop -R
+addons:
+    code_climate:
+        repo_token: YOUR_CODECLIMATE_APP_TOKEN
```

Comment:
I will reformat these lines to have a 2 spaces indentation if you're ok with it @dalpo.